src/items/_menu.scss

```diff
@@ -33,7 +33,7 @@
   align-items: center;
   padding: 0;
 
-  & > div {
+  & > * > div {
     padding: $base-margin / 1.5;
   }
```

Comment:
This could probably be rewritten as:

```
/* As root, not under osjs-panel-item */
.osjs-panel-item-content > div {
  padding: $base-margin / 1.5;
}
```